extension/cart/updateProductsInCart.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @typedef {Object} UpdateCartItem
+ * @property {string} CartItemId
+ * @property {number} quantity
+ */
+
+/**
+ * @param {SDKContext} context
+ * @param {Object} input
+ * @param {UpdateCartItem[]} input.CartItem
+ * @param {function} cb
+ */
+module.exports = function (context, input, cb) {
```

Comment:
Tested this action, but it seems to not work correctly. Changed quantities on update do not change the cart item quantity (no error either).